WordPress/Classes/ViewRelated/Gutenberg/GutenbergNetworking.swift

```diff
@@ -36,27 +37,17 @@ struct GutenbergNetworkRequest {
     // MARK: - Self-Hosed
 
     private func selfHostedRequest(completion: @escaping CompletionHandler) {
-        do {
-            let url = try blog.url(withPath: selfHostedPath).asURL()
-            performSelfHostedRequest(with: url, completion: completion)
-        } catch {
-            completion(.failure(error as NSError))
-        }
+        performSelfHostedRequest(completion: completion)
     }
 
-    private func performSelfHostedRequest(with url: URL, completion: @escaping CompletionHandler) {
-        SessionManager.default.request(url).validate().responseJSON { (response) in
-            switch response.result {
-            case .success(let response):
-                completion(.success(response))
-            case .failure(let error):
-                completion(.failure(error as NSError))
+    private func performSelfHostedRequest(completion: @escaping CompletionHandler) {
+        blog.wordPressOrgRestApi?.GET(path, parameters: nil) { (result, httpResponse) in
+                switch result {
+                    case .success(let response):
+                        completion(.success(response))
+                    case .failure(let error):
+                        completion(.failure(error as NSError))
+                }
             }
-        }
-    }
-
-    private var selfHostedPath: String {
-        let removedEditContext = path.replacingOccurrences(of: "context=edit", with: "context=view")
```

Comment:
I know this is done just for being able to test but I'll drop a comment just not to forget. We can somehow leave requests that are already working with `context=view` out of this solution since the current solution is fragile.

Reply:
@pinar @koke  What about we retry the request with `context=view` if the original one fails?

Reply:
I think I would rather avoid the `context=view` hack completely if we have authentication. It works for the image sizes but won't likely work for other requests and might cause some debugging headaches in the future.